policyengine_us/tests/policy/baseline/gov/states/az/tax/income/credits/az_increased_excise_tax_credit.yaml

```diff
@@ -0,0 +1,169 @@
+- name: AZ increased excise tax credit test 1
+  period: 2023
+  input:
+    state_code: AZ
+    filing_status: HEAD_OF_HOUSEHOLD
+    adjusted_gross_income: 25_001
```

Comment:
these will now only need `az_increased_excise_tax_credit_eligible` and `tax_unit_size`